frontend/src/assets/css/typography-variables.css

```diff
@@ -0,0 +1,74 @@
+:root {
+    --font-weight-400: 400;
+    --font-weight-500: 500;
+    --font-family: 'Rubik', sans-serif;
+
+    /* Heading 1 */
+    --font-size-level-1-heading: 44px;
```

Comment:
base size 16px
```suggestion
    --font-size-level-1-heading: 2.75rem;
```

Reply:
Done in commit [c829865](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/24/commits/c82986598eeb62c53f77120da7f753c700286dd9)